contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -591,6 +591,23 @@ saved. For more details on how to use Merge Patch, see the RFC.
 detailed explanation of how it works and why it needed to be introduced, see
 [here](/contributors/devel/sig-api-machinery/strategic-merge-patch.md).
 
+#### Declaring Patch Strategy and Object Topology
+In the context of [strategic merge patch](/contributors/devel/sig-api-machinery/strategic-merge-patch.md), a set of go markers can be used by API authors to declare the patch strategy of lists, maps and structs (`//+patchStrategy` and `//+patchMergeKey`). Work to move the `apply` functionality to the control plane ([KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/0006-apply.md)) has also introduced markers to describe the topology of lists, maps and structs (`//+listType`, `//+listMapKeys`, `//+mapType`, `//+structType`).
+
+| Marker | Definition |
+|--------|------------|
+| `//+patchStrategy=merge` |  Defines that the merge strategy will be to merge elements of the two lists into one. |
+| `//+patchStrategy=replace` (default)|  Defines that the merge strategy will be touse the list provided in the patch literally rather than merging. |
+| `// +patchMergeKey=<key-name>` |  Applicable in the context where lists are used as maps/dictionaries, i.e. where each list entry is a key-value pair. `patchMergeKey` defines which field should be seen as the key. Combined with `patchStrategy`, this describes what changes should take place on specific elements of a list as part of a patch operation. |
+| `//+listType=atomic/set/map` |  Applicable to objects that are of type list. A single actor can replace the entire list, but cannot change individual elements of it. Unless set to `atomic`, different actors can update individual elements in the list. In that case, the listType can either be `set` (contains scalar elements only) or `map` (contains map-like elements). |
```

Comment:
> Applicable to objects that are of type list. A single actor can replace the entire list, but cannot change individual elements of it. Unless set to `atomic`, different actors can update individual elements in the list.

I find this a little confusing, since you start by saying that a signle actor can replace the entire list.
What about this:
> Applicable to objects that are of type list. Defines if list are owned as a whole by a single actor of if each individual item is owned by a different actor. `atomic` means that the list is owned/changed entirely by a single actor. `set` means that each item in the list can be owned by a different actor, the items of the list must be atomic/scalar. `map` means that the list is an associative list, each actor can own a separate item in the list, the keys have to be scalars.

Reply:
Whoops, I meant to make that line talk about `atomic` only, but didn't do it all the way. Fixed now, I think.